codalab/worker/main.py

```diff
@@ -94,6 +89,12 @@ def main():
         action='store_true',
         help='If specified the worker quits if it finds itself with no jobs after a checkin',
     )
+    parser.add_argument(
+        '--idle-seconds',
+        help='Not running anything for this many seconds constitutes idle',
+        type=int,
+        default=0,
```

Comment:
Is it ok to have a default of 0 for this? Or should we increase the default?

Reply:
This gets overridden by worker manager.  0 makes it backward compatible, and sometimes you want to just run one job with a worker manually and have it quit.